### PR TITLE
Generic docs overhaul

### PR DESCRIPTION
**Pull request by SoldierofOrder**
The goal of this PR is to overhaul the generic guides about Archipelago. It is an early draft that I will continue to add to until I've examined all generic documentation, and I would appreciate as much feedback as y'all are willing to give. Please read below to understand more about my writing.

My target audience is the noob: someone who knows nothing at all about randomizers or Archipelago's inner workings, couldn't tell you the name of any programming language, and is willing to read the glossary when they get confused. Basic mastery of navigating a file explorer and managing .zips is assumed. Anything that requires more advanced knowledge should have a note attached to it remarking that it is for "advanced users", or should walk them through how to do it step-by-step.

My goal with these changes is emphatically **not** to "dumb down" Archipelago's development. My only two goals are to make our communication crystal clear/accessible to a wider audience, and to ease the learning curve for people who'd like to join our community.

To that end, our most necessary tutorials should assume the least about the reader, and anything beyond that should assume that the prerequisite tutorials have also been read. Additionally, lower-level tutorials should point to higher-level tutorials for further reading on relevant topics, to increase site navigability and nudge people to develop a deeper understanding. (For example, all setup guides should assume that the reader has read the generic installation guide for Archipelago, and the generic installation guide should say "Hey, go read the setup guides!")

Please discuss revisions you would like to make to my goals or my work; once I'm done, I intend to extend my grubby little fingers into the non-generic docs, too. I understand that revising all documentation this way is a lot, but with your contributions, this will be a very strong, one-time effort that's less likely to need substantial edits again, and can simply be tweaked as new features arrive or as old features become streamlined.

### CODE REVIEW

**Comment by SoldierofOrder** [comment]
Reopening because the influx of new players has demonstrated that our current docs are insufficient for newbies, and I have a little more time over the next week to make edits. Please offer feedback.

**Comment by SoldierofOrder** [comment]
Thinking of adding screenshots with drawn-on circles to indicate certain elements in the installer. E.g. when describing the Generator, Server, and Clients, showing users exactly what we're referring to. Not sure how to do that, though.

**Comment by Ijwu** [comment]
This looks good so far. I have not done a full on "edit and revision" pass but upon initial perusal it looks like a good set of changes. Please do continue!

File: worlds/generic/docs/using_website_en.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-   such as forfeit rules and the cost for a player to use a hint before generation.
-3. After adjusting the host settings to your liking click on the Upload File button and using the explorer window that
-   opens, navigate to the location where you zipped the player files and upload this zip. The page will generate your
-   game and refresh multiple times to check on completion status.
-4. After the generation completes you will be on a Seed Info page that provides the seed, the date/time of creation, a
-   link to the spoiler log, if available, and links to any rooms created from this seed.
-5. To begin playing, click on "Create New Room", which will take you to the room page. From here you can navigate back
-   to the Seed Info page or to the Tracker page. Sharing the link to the room page with your friends will provide them
-   with the necessary info and files for them to connect to the multiworld.
+First, make sure each player has their config files finished and saved to their computers.
```
Comment (DeerTears) [inline worlds/generic/docs/using_website_en.md:8]: Helps to repeat the same verbs to aid in memory. "Creating a config file"
```suggestion
Make sure each player has their config files created and saved to their computers.
```

**Review by DeerTears — COMMENTED** [review]
I'm no Archipelago vet but I felt like giving this a look-over while this PR is still fresh for clarity. My only big suggestion is helping new users decide if they should self-host or use the website.

**Pull request closed without merging**